sim/druid/moonfire.go

```diff
@@ -0,0 +1,87 @@
+package druid
+
+import (
+	"time"
+
+	"github.com/wowsims/cata/sim/core"
+	"github.com/wowsims/cata/sim/core/proto"
+)
+
+func (druid *Druid) registerMoonfireSpell() {
+	// TODO: Genesis increase to Moonfire duration
+	// TODO: Shooting stars proc on periodic damage
+	// TODO: Glyph of Moonfire increase to periodic damage
```

Comment:
You can split the moonfire spell in 2 (impact and dot) spells and then assign them different class masks to be able to set multis to the dot or the impact only. The spells can use the same spell id with a different tag